src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="form__wrapper">
+      <form action="#"
+      method="get"
+      data-qa="big"
```

Comment:
[checklist](https://github.com/Andriy-Shymkiv/layout_search-bar-airbnb/blob/develop/checklist.md) rule number 5, please fix it everywhere